inst/rmarkdown/templates/cholera_outbreak/skeleton/skeleton.Rmd

```diff
@@ -486,8 +489,10 @@ linelist_cleaned <- linelist_cleaned %>%
 ## OPTIONAL: change those who are above or below a certain age to NA
 # linelist_cleaned <- mutate(linelist_cleaned, 
 #                            age_years = case_when(
-#                              age_years < 0   ~ NA_integer_, 
-#                              age_years > 120 ~ NA_integer_
+#                              is.na(age_years) ~ NA_integer_,
```

Comment:
maybe add this same is.na(age_years) to AJS?  or remove from both just for consistency... [edit: having scrolled down to measles ... just add AJS then all the same]